pkg/version/version.go

```diff
@@ -93,6 +93,14 @@ func (i *Info) IsEqual(other *Info) bool {
 	return i.IsEqualExceptPatch(other) && other.VdbPatch == i.VdbPatch
 }
 
+// Copy copies the parameter's fields into the receiver's
```

Comment:
Why do we need this? There are just ints in this struct. So, Go should be able to make copies of it using regular assignment.